flink-tests/src/test/java/org/apache/flink/test/cancelling/JoinCancelingITCase.java

```diff
@@ -122,9 +122,11 @@ public void testCancelSortMatchWithHighparallelism() throws Exception {
 
 	private static final class SimpleMatcher<IN> implements JoinFunction<Tuple2<IN, IN>, Tuple2<IN, IN>, Tuple2<IN, IN>> {
 		private static final long serialVersionUID = 1L;
+		private static final int WAIT_TIME_PER_RECORD = 300; // 0.3 sec.
```

Comment:
@sunjincheng121 Thanks for reporting and fixing the problem. 
How about make delay configurable in `DelayingMatcher`? Keep `SimpleMatcher` as what it is, i.e, keep it simple. 

Best, Hequn

Reply:
Hi Thanks for the review @hequn8128 !
I think both add the wait_time and make delayingMatcher are can not achieve the purpose of testing。 Because wait_time and delayingMatcher only test the join phase，we want test the sore-merge phase. So we should make sore-merge phase take more time to achieve the purpose of testing. I think a better approach，and will update the PR.